### ISSUE TITLE
what's wrong with radisk-server after yarn add?

### ISSUE THREAD

**Issue by yunsite** [blockstack-radiks/radiks-server#16]
~$ PORT=2319 radiks-server
internal/modules/cjs/loader.js:638
    throw err;
    ^
what's wrong with radisk-server after yarn add?

npm install -g radiks-server
# or, if you use yarn
yarn global add radiks-server

Error: Cannot find module '../index'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/usr/lib/node_modules/radiks-server/bin/radiks-server.js:5:19)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)

**Comment by digitalwaveride** [comment]
Similar issue for me here... after `npm install -g radiks-server` and then `radiks-server` I am getting this... 
```
Error: Cannot find module '../index'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:571:15)
    at Function.Module._load (internal/modules/cjs/loader.js:497:25)
    at Module.require (internal/modules/cjs/loader.js:626:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/radiks-server/bin/radiks-server.js:5:19)
    at Module._compile (internal/modules/cjs/loader.js:678:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:689:10)
    at Module.load (internal/modules/cjs/loader.js:589:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:528:12)
    at Function.Module._load (internal/modules/cjs/loader.js:520:3)
```

Anything I am missing? Following instructions here: https://github.com/blockstack-radiks/radiks-server#built-in-cli-server
Thx!

**Comment by jessicaanhdao** [comment]
I have the same problem...trying to fix

**Comment by jessicaanhdao** [comment]
I had an error that said `Cannot find module '../lib/index'`
So I went back to `radiks-server` directory(mine is in C:\..AppData\Roaming\npm\node_modules\radiks-server or whatever the directory u installed ur radiks-server in), go to `app` directory and fixed the `index.js` file 
changed from `../lib/index.js` on line 5 to `../app/index`
just find out what was import incorrectly from your directory.

**Comment by brunohq** [comment]
> changed from `../lib/index.js` on line 5 to `../app/index`
This worked for me! Thank you, @jessicaanhdao.

**Comment by hstove** [comment]
Sorry for this issue, and thanks to @jessicaandao for the fix! This has been fixed in version `0.2.1`, and `0.2.2` was just published with documentation updates.
